.github/workflows/pkg-pr-new.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish Test Releases
+on: [push, pull_request]
```

Comment:
a) do you want this on every push?
b) what's the permissino story here? does this use secrets or not? will it work on forks?

Reply:
a) I think so? That way it stays up to date through the life of a branch
b) no secrets, so I *think* it should work on forks. cc @AmirSa12

Reply:
a) You can do it the vite way where it only run on specific comments on a PR! But having things on each push would be good! what you might want to do is to run it on each push on main and only on approved PRs. 
b) yep, no secrets, but for forks, the user has to enable workflows for their fork plus having the app installed on the repo. So it's a matter of enabling workflows.

Reply:
To avoid workflow collisions, you can also do this! https://github.com/stackblitz-labs/pkg.pr.new/blob/main/.github/workflows/ci.yml#L2-L7 since the push and pull_request event might run two times at the same time